toon.py

```diff
@@ -12,31 +12,49 @@
 import homeassistant.helpers.config_validation as cv
 
 # Home Assistant depends on 3rd party packages for API specific code.
-REQUIREMENTS = ['toonlib==0.4.0']
+REQUIREMENTS = ['toonlib==0.5.0']
 
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = 'toon'
 TOON_HANDLE = 'toon_handle'
+CONF_GAS = 'gas'
+DEFAULT_GAS = True
+CONF_SOLAR = 'solar'
+DEFAULT_SOLAR = False
```

Comment:
So the idea here is that you can use this setting to override gaps in the initial json response. Setting this to True would always show Solar even if you don't have any solar panels. It's not very pretty, but I also don't want people to need to restart ToonHA for 5 times before all the components load